docs/MIGRATION_REVIEW.md

```diff
@@ -0,0 +1,231 @@
+   - A hybrid approach based on domain?
+
+### Historical & Time-Sensitive Data
+
+9. **Tournament History (Phase 4)**  
+   Should historical MRM tournament data be migrated, or only the current/upcoming tournament structure? Historical data could be valuable for archives but adds complexity.
+
+10. **Year End Poll Historical Data (Phase 5)**  
+    The Year End Poll spans multiple years. Should each year be modeled separately, or should there be a year-scoped structure in Sanity?
+
+### Technical Implementation
+
+11. **Migration Rollback Strategy**  
+    Is there a rollback plan if a migration phase fails or corrupts data? This could involve:
+    - Sanity dataset snapshots before each phase
+    - MySQL backup verification
+    - Dry-run capability in migration scripts
+
+12. **Incremental vs. Full Migration**  
+    Will migrations be one-time full imports, or do you need support for incremental/delta migrations for ongoing updates during the transition period?
+
+13. **Environment Strategy**  
+    What's the plan for staging/testing migrations? Will there be:
+    - A separate Sanity dataset for testing (e.g., `staging` vs `production`)?
+    - Test migrations against a copy of production data?
+
+---
+
+## Suggestions
+
+### Documentation Improvements
+
+1. **Add Migration Status Tracking**  
+   Consider adding a `status` field to each phase in `index.yml`:
+   ```yaml
+   - phase: 1
+     name: "Basic Models"
+     status: "in_progress"  # not_started | in_progress | completed | blocked
+     models:
+       - name: Deejay
+         status: completed
+       - name: Ad
+         status: not_started
+   ```
+
+2. **Create a Migration Checklist**  
+   For each model, document:
+   - [ ] Schema designed in Sanity
+   - [ ] Migration script created
+   - [ ] Test migration completed
+   - [ ] Data validated
+   - [ ] Production migration completed
+   - [ ] PHP code updated to use Sanity
+
+### Technical Suggestions
+
+3. **Shared Migration Utilities**  
+   Create a `bin/migrations/shared/` folder with:
+   - `imageUploader.ts` - Reusable image asset upload logic
+   - `richTextConverter.ts` - HTML to Portable Text conversion
+   - `validation.ts` - Data validation utilities
+   - `logger.ts` - Consistent logging across all migrations
+
+4. **Dry Run Mode**  
+   Add a `--dry-run` flag to migration scripts that:
+   - Validates source data
+   - Shows what would be migrated
+   - Doesn't write to Sanity
+   - Generates a preview report
+
+5. **Migration Reports**  
+   Generate post-migration reports similar to `DEEJAY_MIGRATION_REPORT.md` for each model, documenting:
+   - Records processed
+   - Records skipped (with reasons)
+   - Validation errors encountered
+   - Image assets uploaded
+
+6. **Consider a Phased Frontend Approach**  
+   Instead of a big-bang frontend migration, consider:
+   - Phase A: Read from Sanity, fall back to MySQL
+   - Phase B: Write to both Sanity and MySQL (dual-write)
```

Comment:
I'm of the opinion that dual write is too complex and that we should just keep running incremental migrations until we have full parity, then cut over